doc/en/api/1.0.0/importer.yaml

```diff
@@ -99,14 +127,585 @@ paths:
         204:
           description: No Content.
 
+parameters:
+  AsyncParameter:
+    name: async
+    description: Run the import asyncronously. Defaults to false.
```

Comment:
you can document a default value, no need to waste description on it.

And that way the request generator can move faster.